Makefile

```diff
@@ -79,7 +81,11 @@ lint:
 presto-server-rpm.rpm:
 	wget 'https://repository.sonatype.org/service/local/artifact/maven/content?r=central-proxy&g=com.facebook.presto&a=presto-server-rpm&e=rpm&v=RELEASE' -O $@
 
-smoke: clean-test-all test-images
+smoke: clean-test-all test-images _smoke
+
+smoke-configurable-cluster: clean-test-all _smoke
```

Comment:
Or actually I can move the `rm -rf tmp` to `clean-test`. I think that should be better than introducing yet another clean target. Thoughts?